shiny/render/renderer/_renderer.py

```diff
@@ -203,23 +255,39 @@ def __init__(
 
     async def transform(self, value: IT) -> Jsonifiable:
         """
-        Renderer - transform docs here
-
-        TODO-barret-docs
+        Transform an non-`None` output value into a JSON-serializable object.
+
+        If an `.render()` method is not implemented, this method **must** be
```

Comment:
I think it would be good to start out by explaining that, for most situations, you don't need to implement `.transform()` (and you should implement `.render()` instead?)

Reply:
It's the other way around. 

90%+ of the time you should probably implement the transform method. The other 10%, you'll need to implement the render method.